Libraries/LibC/stdio.cpp

```diff
@@ -415,6 +417,18 @@ long FILE::tell()
     return lseek(m_fd, 0, SEEK_CUR);
 }
 
+void FILE::reopen(int fd, int flags)
+{
+    // Should these failing fail something?
```

Comment:
Dr. POSIX says "Failure to flush or close the file descriptor successfully shall be ignored. The error and end-of-file indicators for the stream shall be cleared.", so no.

(Change the comment to state that we ignore errors intentionally.)

Reply:
Dr. POSIX is just a suggestion at this point, I thought we might want something so I left that comment there _specifically_ for you :)

Reply:
Sure, it is just a suggestion, but in this case it makes sense for us to take its suggestion :)